politeiawww/cmd/politeiawww_dbutil/README.md

```diff
@@ -1,35 +1,132 @@
 # politeiawww_dbutil
 
 politeiawww_dbutil is a tool that allows you to interact with the politeiawww
-database.
+database. 
+**Note**: The politeiawww database works with LevelDB and CockroachDB. 
+If you are using LevelDB you have to shut down politeiawww before using this tool.
 
-**Note**: You currently have to shut down politeiawww before using this tool.
 
+## Available Commands
+You can view the available commands and application options by using the help
+flag.
 
-## Usage
+`$ politeiawww_dbutil -h`
 
-You can specify the following options:
+You can view details about a specific command, including required arguments,
+by putting the help flag after the command.
 
+`$ politeiawww_dbutil <command> -h`
+
+
+## Setup Configuration File
```

Comment:
There is an inconsistent naming convention between OS's here.  The macOS is putting the conf file in `politeiawww/dbutil` where as windows and ubuntu put it in `politeiawww/cli`.

More generally though, `politeiawww_dbutil` conf and data should probably not be part of the politeiawww directory.

`osDataDir/politeiawww_dataload` instead of `osDataDir/politeiawww/dbutil`